src/layouts/default.vue

```diff
@@ -92,14 +87,20 @@ const embeddedPage = {
       isSidebarVisible,
       isSearchRoute,
       headerHasTwoRows,
-      mainContentRef,
-      mainRef,
     }
   },
 }
 export default embeddedPage
 </script>
+
 <style scoped>
+.sidebar {
```

Comment:
Can we add the CSS variables for banner height here?

Reply:
It actually shouldn't need those to work correctly anymore, but if you notice that we need them in testing please let me know.